docs/getting-started/generated-app.md

```diff
@@ -36,24 +40,41 @@ The generated service is built with the following proven open-source technologie
 - [Jest](https://jestjs.io/) - delightful JavaScript Testing Framework with a focus on simplicity.
 - [Docker](https://www.docker.com/) - an open platform for developing, shipping, and running applications.
 
+#### .NET
+
+- [ASP.NET Core](https://dotnet.microsoft.com/en-us/apps/aspnet) - A framework for building internet-connected, web-based applications.
+- [Entity Framework Core](https://docs.microsoft.com/en-us/ef/core/) - A modern object-database mapper for .NET. It supports queries, change tracking, updates, and schema migrations.
+- [SQL Server](https://www.microsoft.com/en-us/sql-server) - A relational database management system developed by Microsoft.
+- [IdentityServer](https://identityserver.io/) - An OpenID Connect and OAuth 2.0 framework for ASP.NET Core, enabling you to serve and authenticate API requests securely.
+- [XUnit](https://xunit.net/) - A free, open source, community-focused unit testing tool for the .NET Framework.
+- [Swagger](https://swagger.io/tools/swagger-ui/) - A Swagger tool implementation for API's built with ASP.NET Core, generating beautiful API documentation including a UI to explore and test operations, directly from your routes, controllers, and models.
+- [Docker](https://www.docker.com/) - Same as above, used for creating, deploying, and running applications using containers.
+
 ### Client-Side
 
 - [React](https://reactjs.org/) - A JavaScript library for building user interfaces.
 - [React-Admin](https://marmelab.com/react-admin/) - A frontend Framework for building B2B applications running in the browser on top of REST/GraphQL APIs, using ES6, React and Material Design.
 
 ## Projects Structure
 
-The generated service is built from two projects, each in a separate folder
+The code for your generated service is built from two projects, each in a separate folder, `server`, and an optional `admin`:
```

Comment:
@mulygottlieb - Just making sure it's accurate also for the .NET structure.

Reply:
@PazYanoverr @dericksozo I *think* we will start at least in the first phase without an AdminUI for the .NET projects. We need to discuss what this also means for our UI where today you can choose if you want Admin UI and if you want GraphQL (you must choose GQL if you want the AdminUI) and I think the first .NET release will be REST only so no GQL and therefore no Admin UI neither. 
I think that once we add the GQL to .NET DSG, we can consider adding Admin UI for .NET as well (the AdminUI we generate is in React anyway so we need to think if it makes sense + is anyone really using it)

Reply:
@PazYanoverr @dericksozo I grabbed a quick chat with Yuval. Let's indeed start in .NET without Admin UI at all and we will add it down the line according to priority. So for now we can remove the concept of "Admin" from .NET-related documentation.

Reply:
Good point, thanks @mulygottlieb!
@dericksozo - Let's update the documentation to reflect it. I don't think we need to say that in .NET we do not support GQL and admin UI, just to talk about these in the context of node.js only.